sled-agent/src/instance.rs

```diff
@@ -53,7 +53,7 @@ pub enum Error {
 
     // TODO: Remove this error; prefer to retry notifications.
     #[error("Notifying Nexus failed: {0}")]
-    Notification(nexus_client::Error<()>),
+    Notification(nexus_client::Error<nexus_client::types::Error>),
```

Comment:
Can you explain this change?

Reply:
Dropshot now emits the `Error` type in the OpenAPI doc. Progenitor picks it up, so that the return type for all methods is `Result<T, progenitor_client::Error<***_client::types::Error>>`. This changes the sim interfaces to match.